model/project.go

```diff
@@ -1441,28 +1467,55 @@ func (p *Project) FindAllBuildVariantTasks() []BuildVariantTaskUnit {
 	allBVTs := []BuildVariantTaskUnit{}
 	for _, b := range p.BuildVariants {
 		for _, t := range b.Tasks {
-			tasksToPopulate := []ProjectTask{}
+			t.Variant = b.Name
 			if t.IsGroup {
 				group, exists := taskGroups[t.Name]
 				if !exists {
 					continue
 				}
-				for _, groupTask := range group.Tasks {
-					tasksToPopulate = append(tasksToPopulate, tasksByName[groupTask])
-				}
+				allBVTs = append(allBVTs, p.TasksFromGroup(t, group)...)
 			} else {
-				tasksToPopulate = append(tasksToPopulate, tasksByName[t.Name])
-			}
-			for _, pTask := range tasksToPopulate {
-				t.Populate(pTask)
-				t.Variant = b.Name
+				t.Populate(tasksByName[t.Name])
 				allBVTs = append(allBVTs, t)
 			}
 		}
 	}
 	return allBVTs
 }
 
+// TasksFromGroup returns a slice of the task group's tasks. groupTask is merged with each of the tasks in the group.
```

Comment:
Can this comment specify that anything not specified in the task group definition groupTask is populated from the task definition (as opposed to "merge" which feels ambiguous)

Reply:
done